tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -131,6 +132,11 @@ struct dispatch_constants {
     uint32_t prefetch_d_buffer_pages_;
 };
 
+// Define a queue type, for when they're interchangeable.
+#define CQ_COMPLETION_QUEUE true
+#define CQ_ISSUE_QUEUE false
+typedef bool cq_queue_t;
```

Comment:
this is sort of odd, not clear why CQ_COMPLETION_QUEUE would be a boolean.  More explanation

Reply:
Originally I was just using a bool, then added a type, but yeah it should have been an enum. Updated this & moved it to the debug_tools file where it's used.